src/redux/books/booksSlice.js

```diff
@@ -1,40 +1,52 @@
-const ADD_BOOK = 'bookstore/books/ADD_BOOK';
-const REMOVE_BOOK = 'bookstore/books/REMOVE_BOOK';
+import axios from 'axios';
+import { createAsyncThunk } from '@reduxjs/toolkit';
 
-const initialState = [
-  {
-    id: 1,
-    title: 'The Hunger Game',
-    author: 'Suzanne Collins',
-  },
-  {
-    id: 2,
-    title: 'Dune',
-    author: 'Frank Herbert',
-  },
-  {
-    id: 3,
-    title: 'Capital in the Twenty-First Century',
-    author: 'Suzanne Collins',
-  },
-];
+const ADD_BOOK = 'ADD_BOOK';
+const REMOVE_BOOK = 'REMOVE_BOOK';
+const GET_BOOKS = 'GET_BOOKS';
+const URL = 'https://us-central1-bookstore-api-e63c8.cloudfunctions.net/bookstoreApi/apps/Hhnw4p1fH7cbvjMY3ewX/books';
 
-export const addbook = (book) => ({
-  type: ADD_BOOK,
-  payload: book,
+const initialState = [];
+
+export const addbook = createAsyncThunk(ADD_BOOK, async (book) => {
+  await axios.post(URL, {
+    item_id: book.id,
+    title: book.title,
+    author: book.author,
+    category: book.category,
+  });
```

Comment:
- [ ] Because your UI does not have any input for category, so this return an error that category is required when trying to add a book, kindly fix that
![Screenshot from 2023-04-05 18-09-03](https://user-images.githubusercontent.com/66097371/230154239-a4e448c2-7399-49af-b35f-8e79135f2e8c.png)